README.md

```diff
@@ -49,43 +49,35 @@ pip install git+https://github.com/childmindresearch/wristpy
 
 ## Quick start
 
-Here is a sample script that goes through the various functions that are built into wristpy. 
-
-```Python
-
-#loading the prerequisite modules
-import wristpy
-from wristpy.core import computations
-from wristpy.io.readers import readers
-from wristpy.processing import metrics, analytics
-
-#set the paths to the raw data and the desired output path
-file_path = '/path/to/your/file.gt3x'
-
-#load the acceleration data
-test_data = readers.read_watch_data(file_path)
-
-#calibrate the data
-calibrator = calibration.Calibration()
-calibrated_data = calibrator.run(test_data.acceleration)
-
-#Compute some metrics and get epoch1 data
-enmo = metrics.euclidean_norm_minus_one(calibrated_data)
-anglez = metrics.angle_relative_to_horizontal(calibrated_data)
-
-enmo_epoch1 = computations.moving_mean(enmo)
-anglez_epoch1 = computations.moving_mean(anglez)
-
-#Find sleep windows
-sleep_detector_class = analytics.GGIRSleepDetection(anglez)
-sleep_windows = sleep_detector_class.run_sleep_detection()
+### Using Wristpy through the command-line:
+```sh
+python wristpy /input/file/path.gt3x -o /save/path/file_name.csv -c gradient
```

Comment:
It should be possible to run it as just "wristpy" if it's installed in the current environment.